cfgov/core/management/commands/generate_streamfield_docs.py

```diff
@@ -0,0 +1,127 @@
+from collections import defaultdict
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from wagtail.wagtailcore.blocks import StreamBlock
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailcore.models import Page
+
+
+class Command(BaseCommand):
+    help = 'Generate Wagtail StreamField block documentation'
+
+    def handle(self, *args, **options):
+        blocks, models = get_blocks_and_models()
+
+        template = loader.get_template('core/docs/streamfield-blocks.md')
```

Comment:
Thinking ahead, it might be nice if this command took the template path as an input, rather than hard-coding it. Right now with the template file in `cfgov/core/docs`, we end up with some of our documentation (the wording around the templated content) in a non-obvious location well outside of `docs/`.

Reply:
Where would you propose this template live instead? I considered putting it somewhere like `docs/templates` but the template is really only useful in the context of the management command. Additionally, the template intro comment specifically cites the management command as the thing to run.

Reply:
If this page stood alone from the rest of the Markdown MkDocs pages (had a different style, etc) it'd be easier to package the template/command independently. Given that we want the built page to live in the source repository under `docs/` among the other pages, though, I'm not sure what the best separation would be.

Originally (in [this branch on my fork](https://github.com/cfpb/cfgov-refresh/compare/master...chosak:dynamic-streamfield-docs#diff-ea4000de77ab8d50011836372a0fe126)) I tried to separate a bit more by having the management command just generate the JSON and then have MkDocs load the JSON at render time. This would keep the document content/layout in `docs/` and limit the Django interaction with generating source data. My preference is still towards that approach, but I got some feedback that people instead preferred having the fully rendered doc in the repo, so went in the direction in this PR.

Reply:
If you're referring to me, I don't think I meant to say that. I'm not sure I even realized that an MkDocs page _could_ load JSON at render time. I think that's fine, actually. I just wanted the _data_ to be snapshotted at the point in time when stuff is merged to master, so it's always accurate with that.

Reply:
I don't know and don't feel like I have enough time to think it through, to be honest. I also like the idea of being able to package this separately, as @willbarton suggests, but we've gotta focus on MVPs as we are approaching the finish line of this project. Do what you think is best 😀

Reply:
Honestly, my experience with the way the mini projects happen, if it's not a separate package *now*, it'll never be. See https://github.com/cfpb/wagtail-treemodeladmin vs https://github.com/cfpb/wagtail-treemodeladmin